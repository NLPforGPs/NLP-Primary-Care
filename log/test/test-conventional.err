cpu-bind=MASK - bp1-gpu012, task 4294967295 4294967295 [0]: mask 0x101 set

CondaValueError: prefix already exists: /user/home/zu20361/.conda/envs/NLP_GP

[nltk_data] Downloading package wordnet to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package wordnet is already up-to-date!
[nltk_data] Downloading package punkt to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
Some weights of the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext were not used when initializing BertForSequenceClassification: ['cls.predictions.transform.LayerNorm.bias', 'cls.predictions.decoder.weight', 'cls.predictions.transform.LayerNorm.weight', 'cls.seq_relationship.weight', 'cls.predictions.decoder.bias', 'cls.predictions.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.dense.bias', 'cls.seq_relationship.bias']
- This IS expected if you are initializing BertForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of BertForSequenceClassification were not initialized from the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext and are newly initialized: ['classifier.bias', 'classifier.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
2021-12-09 22:42:05,404 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:155] - INFO: Predicting...
2021-12-09 22:42:05,449 - /user/home/zu20361/.conda/envs/NLP_GP/lib/python3.9/site-packages/datasets/builder.py[line:377] - WARNING: Using custom data configuration default-135fbc7b45d5ff11
0 examples [00:00, ? examples/s]2021-12-09 22:42:05,653 - /user/home/zu20361/.cache/huggingface/modules/datasets_modules/datasets/transcript_evaldataset/9a9f33ed1951859e6f0a536ab67cad76ef5b13196ce8d7b688334d25b0f51aa6/transcript_evaldataset.py[line:60] - INFO: Generating examples...
130 examples [00:00, 1299.18 examples/s]                                          0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00, 401.60it/s]
2021-12-09 22:42:05,811 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:212] - INFO: conventional classifier...
Some weights of the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext were not used when initializing BertForSequenceClassification: ['cls.predictions.transform.LayerNorm.bias', 'cls.predictions.decoder.weight', 'cls.predictions.transform.LayerNorm.weight', 'cls.seq_relationship.weight', 'cls.predictions.decoder.bias', 'cls.predictions.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.dense.bias', 'cls.seq_relationship.bias']
- This IS expected if you are initializing BertForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of BertForSequenceClassification were not initialized from the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext and are newly initialized: ['classifier.bias', 'classifier.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
2021-12-09 22:42:08,232 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:224] - INFO: id to label is {0: 'blood', 1: 'endocrine', 2: 'general', 3: 'psychological', 4: 'male', 5: 'skin', 6: 'pregnancy', 7: 'female', 8: 'cardiovascular', 9: 'ear', 10: 'digestive', 11: 'urological', 12: 'neurological', 13: 'musculoskeletal', 14: 'eye', 15: 'respiratory'}
  0%|          | 0/1 [00:00<?, ?ba/s]100%|██████████| 1/1 [00:01<00:00,  1.87s/ba]100%|██████████| 1/1 [00:01<00:00,  1.87s/ba]
  0%|          | 0/2 [00:00<?, ?ba/s] 50%|█████     | 1/2 [00:00<00:00,  1.22ba/s]100%|██████████| 2/2 [00:01<00:00,  1.95ba/s]100%|██████████| 2/2 [00:01<00:00,  1.79ba/s]

real	0m40.205s
user	0m22.300s
sys	0m9.753s
