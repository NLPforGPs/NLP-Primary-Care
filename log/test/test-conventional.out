JOB ID: 272461
Working Directory: /user/home/zu20361/NLP-Primary-Care
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Start Time: Thu Dec 9 22:41:51 GMT 2021
loading codes from PE...
Downloading and preparing dataset description_dataset/default (download: Unknown size, generated: 2.07 MiB, post-processed: Unknown size, total: 2.07 MiB) to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-135fbc7b45d5ff11/0.0.0/9a9f33ed1951859e6f0a536ab67cad76ef5b13196ce8d7b688334d25b0f51aa6...
Dataset description_dataset downloaded and prepared to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-135fbc7b45d5ff11/0.0.0/9a9f33ed1951859e6f0a536ab67cad76ef5b13196ce8d7b688334d25b0f51aa6. Subsequent calls will reuse this data.
classification_report:
                 precision    recall  f1-score   support

          blood       0.21      0.75      0.32         8
 cardiovascular       0.56      0.72      0.63        32
      digestive       0.68      0.68      0.68        44
            ear       0.89      0.73      0.80        11
      endocrine       0.35      0.88      0.50        24
            eye       0.25      0.40      0.31         5
         female       0.57      0.57      0.57        14
        general       0.03      0.29      0.06        14
           male       0.38      0.71      0.50         7
musculoskeletal       0.74      0.85      0.79        65
   neurological       0.46      0.60      0.52        20
      pregnancy       1.00      0.09      0.17        11
  psychological       0.34      0.84      0.48        50
    respiratory       0.66      0.89      0.76        37
           skin       0.65      0.69      0.67        32
     urological       0.76      0.72      0.74        18

      micro avg       0.43      0.73      0.54       392
      macro avg       0.53      0.65      0.53       392
   weighted avg       0.57      0.73      0.61       392
    samples avg       0.46      0.76      0.54       392

f1_score 0.5313569834314968
roc_auc 0.5546484808299343
End Time: Thu Dec 9 22:42:32 GMT 2021
